infrastructure-examples/aws/machines-v2.yml

```diff
@@ -47,7 +47,8 @@ aws:
       tags:
         type: dbt2-driver
     pg1:
-      spot_max_price: .50
+      spot: true
+      spot_instance_interruption_behavior: terminate
```

Comment:
This needs to be updated to `stop` or moved under `dbt2-driver` since `terminate` causes issues with additional ebs volumes.

Reply:
done.